.github/dependabot.yaml

```diff
@@ -3,13 +3,13 @@ updates:
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
-      interval: "weekly"
+      interval: "monthly"
```

Comment:
Not sure what the default is, probably best to use an explicit `day: "monday"`.

Reply:
Monday is the default, yeah.  Changed to state it explicitly though.